SR2024/2023-10-17-kickstart-reminder.md

```diff
@@ -0,0 +1,65 @@
+---
+to: Student Robotics 2024
+subject: Getting the most out of SR2024 Kickstart
+---
+
+Hello there!
+
+With [Kickstart][kickstart] this Saturday we wanted to let you know what to
+expect on the day and how to get the best start on your robot.
+
+As you already know, the event is fully remote. There are two main aspects to
+the day:
+
+- the presentation, which will be [livestreamed][livestream] on Youtube, and
+- the microgames, a set of tasks to familiarise you with the kit (details on
+  what you'll need below)
+
+We encourage all teams to watch the livestream on the day and work through the
+microgames together afterwards. If you are unable to do this for some reason the
+presentation and microgames will continue to be available afterwards on the
+[Kickstart event page][kickstart] and remain a great place for your team to
+start working on their robot.
+
+There will be two document links for the microgames -- one for the competitors
+which outlines the challenges and another for you which has the answers.
+
+What you'll need for the microgames:
+
+- **your kit**; all the kits have been dispatched -- please let us know if yours
+  hasn't arrived!
+
+- at least one **computer** that can install and run the [simulator][simulator];
+  we'd encourage installing the Webots software ahead of time, though note that
+  you will need to download the simulation files afresh from our docs after the
+  game has been announced at Kickstart.
```

Comment:
I'd really like to get the improved overview page from https://github.com/srobo/docs/pull/540 merged before sending this. If not, we should probably link directly to the setup page rather than the overview page (since the overview is currently very lacking in content).

Reply:
Related PR has now merged.